src/diffusers/models/resnet.py

```diff
@@ -446,10 +446,17 @@ def upfirdn2d_native(input, kernel, up=1, down=1, pad=(0, 0)):
     kernel_h, kernel_w = kernel.shape
 
     out = input.view(-1, in_h, 1, in_w, 1, minor)
+
+    # Temporary workaround for mps specific issue: https://github.com/pytorch/pytorch/issues/84535
+    using_mps = out.device.type == "mps"
```

Comment:
think this doesn't really warrant a new variable -> it now forces me to look up what `using_mps` means every time I see it